docs/en/latest/how-to-build.md

```diff
@@ -59,7 +61,7 @@ This installation method is suitable for CentOS 7, please run the following comm
 sudo yum install -y https://github.com/apache/apisix/releases/download/2.10.1/apisix-2.10.1-0.el7.x86_64.rpm
```

Comment:
Can we only keep the installation from our rpm repo? There is no need to submit the rpm to GitHub anymore.

Reply:
@spacewander agreed. Done. Thanks.